__tests__/app.test.js

```diff
@@ -103,3 +103,48 @@ describe("GET /api/articles/:id", () => {
       });
   });
 });
+
+describe("GET /api/articles/:id/comments", () => {
+  test("200: responds with an array of comments", () => {
+    return request(app)
+      .get("/api/articles/9/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments.length).toBe(2);
+        comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            article_id: 9,
+          });
+        });
+      });
+  });
+});
+
+describe("GET /api/articles/:id", () => {
+  test("404: responds with an empty object", () => {
```

Comment:
when does it respond with an empty object? Can you add more to the test description?